frameworks/Scala/scruffy/src/test/scala/BenchmarkVerify.scala

```diff
@@ -11,7 +11,7 @@ object BenchmarkVerify extends App {
 
   val sw = Stopwatch()
 
-  val futures = for ( k <- 1 to 10000 ) yield {
+  val futures = for ( k <- 1 to 30000 ) yield {
```

Comment:
Correct. It's not even included in the package that is built when you build the jar. It's just a smoke test I run when I upgrade the version of Scruffy used by the tests. It's typical in Java/Scala to keep the test code in a source folder called src/test and the main code in src/main.